OpenRA.Mods.Common/Widgets/ScrollPanelWidget.cs

```diff
@@ -64,6 +64,8 @@ public class ScrollPanelWidget : Widget
 		// The current value is the actual list offset at the moment
 		float currentListOffset;
 
+		public string ClickSound = ChromeMetrics.Get<string>("ClickSound");
```

Comment:
Please put this up top with the other fields that are exposed to yaml.  Probably best to keep consistency with the other fields and omit the `readonly` (a future PR can fix all the widgets properly).